packages/account/src/Components/Routes/__tests__/binary-link.spec.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Router } from 'react-router-dom';
+import { createBrowserHistory } from 'history';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { PlatformContext } from '@deriv/shared';
+import { findRouteByPath } from '../helpers';
+import BinaryLink from '../binary-link.jsx';
+
+jest.mock('Stores/connect', () => ({
+    __esModule: true,
+    default: 'mockedDefaultExport',
+    connect: () => Component => Component,
+}));
+
+jest.mock('../helpers.js', () => ({
+    findRouteByPath: jest.fn(() => '/test/path'),
+    normalizePath: jest.fn(() => '/test/path'),
+}));
+
+jest.mock('Constants/routes-config.js', () => () => ({
+    getRoutesConfig: jest.fn(() => []),
+}));
+
+describe('<BinaryLink />', () => {
+    const history = createBrowserHistory();
+
+    it('should show and trigger Navlink with path and active className', () => {
+        render(
+            <PlatformContext.Provider value={{ is_appstore: false }}>
+                <Router history={history}>
+                    <BinaryLink to='test-link'>Simple test link</BinaryLink>
+                </Router>
+            </PlatformContext.Provider>
+        );
+
+        expect(screen.getByText('Simple test link')).toBeInTheDocument();
+
+        fireEvent.click(screen.getByRole('link'));
+        expect(window.location.pathname).toBe('/test/path');
+    });
+
+    it('should show simple link text', () => {
+        render(
+            <PlatformContext.Provider value={{ is_appstore: false }}>
+                <Router history={history}>
+                    <BinaryLink>Simple test link without Navlink</BinaryLink>
+                </Router>
+            </PlatformContext.Provider>
+        );
+
+        expect(screen.getByText('Simple test link without Navlink')).toBeInTheDocument();
+    });
+    it('should show simple link text', () => {
```

Comment:
can u give a different description for this? it is the duplication of previous one.